mtags-java/src/main/scala/scala/meta/internal/pc/JavaPresentationCompiler.scala

```diff
@@ -0,0 +1,191 @@
+
+  override def getTasty(
+      targetUri: URI,
+      isHttpEnabled: Boolean,
+  ): CompletableFuture[String] = CompletableFuture.completedFuture("")
+
+  override def autoImports(
+      name: String,
+      params: OffsetParams,
+      isExtension: lang.Boolean,
+  ): CompletableFuture[util.List[AutoImportsResult]] =
+    CompletableFuture.completedFuture(Nil.asJava)
+
+  override def implementAbstractMembers(
+      params: OffsetParams
+  ): CompletableFuture[util.List[TextEdit]] =
+    CompletableFuture.completedFuture(Nil.asJava)
+
+  override def insertInferredType(
+      params: OffsetParams
+  ): CompletableFuture[util.List[TextEdit]] =
+    CompletableFuture.completedFuture(Nil.asJava)
+
+  override def extractMethod(
+      range: RangeParams,
+      extractionPos: OffsetParams,
+  ): CompletableFuture[util.List[TextEdit]] =
+    CompletableFuture.completedFuture(Nil.asJava)
+
+  override def convertToNamedArguments(
+      params: OffsetParams,
+      argIndices: util.List[Integer],
+  ): CompletableFuture[util.List[TextEdit]] =
+    CompletableFuture.completedFuture(Nil.asJava)
+
+  override def didChange(
+      params: VirtualFileParams
+  ): CompletableFuture[util.List[Diagnostic]] =
+    CompletableFuture.completedFuture(Nil.asJava)
+
+  override def didClose(uri: URI): Unit = ()
+
+  override def semanticdbTextDocument(
+      filename: URI,
+      code: String,
+  ): CompletableFuture[Array[Byte]] =
+    CompletableFuture.completedFuture(Array.emptyByteArray)
+
+  override def selectionRange(
+      params: util.List[OffsetParams]
+  ): CompletableFuture[util.List[SelectionRange]] =
+    CompletableFuture.completedFuture(Nil.asJava)
+
+  override def shutdown(): Unit = ()
+
+  override def restart(): Unit = ()
+
+  override def withSearch(search: SymbolSearch): PresentationCompiler =
+    copy(search = search)
+
+  override def withExecutorService(
+      executorService: ExecutorService
+  ): PresentationCompiler =
+    copy(ec = ExecutionContext.fromExecutorService(executorService))
+
+  override def withScheduledExecutorService(
+      scheduledExecutorService: ScheduledExecutorService
+  ): PresentationCompiler = copy(sh = Some(scheduledExecutorService))
+
+  override def withConfiguration(
+      config: PresentationCompilerConfig
+  ): PresentationCompiler = copy(config = config)
+
+  override def withWorkspace(workspace: Path): PresentationCompiler =
+    copy(workspace = Some(workspace))
+
+  override def newInstance(
+      buildTargetIdentifier: String,
+      classpath: util.List[Path],
+      options: util.List[String],
+  ): PresentationCompiler =
+    copy(
+      buildTargetIdentifier = buildTargetIdentifier,
+      classpath = classpath.asScala,
+      options = options.asScala.toList,
+    )
+
+  override def diagnosticsForDebuggingPurposes(): util.List[String] = Nil.asJava
+
+  override def isLoaded: Boolean = true
+
+  override def scalaVersion(): String = BuildInfo.scalaCompilerVersion
```

Comment:
Can we omit this method?
I think it only makes sense for scala pc

Reply:
I dont really want to change PC interface

Reply:
Then I would replace its value from `BuildInfo.scalaCompielrVersion` by hardcoded value `"java"`.
+ delete buildInfo plugin and setting from `mtags-java`